pkg/resources/helpers_test.go

```diff
@@ -6,10 +6,10 @@ import (
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
 	"github.com/stretchr/testify/require"
 
-	"github.com/Snowflake-Labs/terraform-provider-snowflake/pkg/provider"
 	"github.com/Snowflake-Labs/terraform-provider-snowflake/pkg/resources"
 )
 
+// todo: remove the rest of these which are not used. also this file should be renamed for clarity to make it clear it is for testing only
```

Comment:
Can we have a jira to handle this, so that this is not left hanging here. You can add it e.g. to SNOW-926148 and put the number here in the comment